<ui version="4.0" >
 <class>Kleo::Dialogs::CertifyCertificateDialogPrivate::SelectCheckLevelWidget</class>
 <widget class="QWidget" name="Kleo::Dialogs::CertifyCertificateDialogPrivate::SelectCheckLevelWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>959</width>
    <height>606</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="2" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>&lt;b>Step 2:&lt;/b> How thoroughly have you checked that this key actually belongs to the person it claims it's from?</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QRadioButton" name="checkLevelNotCheckedRB" >
     <property name="text" >
      <string>I have not checked at all</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Describe semantics here...</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="QRadioButton" name="checkLevelCasualRB" >
     <property name="text" >
      <string>I have checked casually</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1" >
    <widget class="QLabel" name="label_4" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>Choose this if you know the person, and have asked him whether this was his certificate, but you have not verified he is actually in posession of the secret certificate.</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2" >
    <widget class="QRadioButton" name="checkLevelThoroughlyRB" >
     <property name="text" >
      <string>I have checked very thoroughly</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="1" >
    <widget class="QLabel" name="label_5" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>Choose this if you have e.g. checked the ID card of the person, and have verified that he is in posession of the secret key (e.g. by sending encrypted mails to him and asking him to decrypt them).</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
